views/js/checkout-component-renderer.js

```diff
@@ -223,10 +223,13 @@ function renderPaymentMethods() {
             }
 
             if (!placeOrderAllowed) {
-                component.showValidation();
+                if (!!component) {
```

Comment:
```suggestion
                if (component && component.showValidation) {
```

Reply:
The show validation is a function of every component, is it needed to check in the if `component.showValidation` as well?